src/config.cpp

```diff
@@ -464,6 +473,32 @@ void parse_program_options(int argc, const char** argv)
                 std::exit(EXIT_FAILURE);
             }
         }
+
+        if (arguments.provided("syscalls"))
+        {
+            config.use_syscalls = true;
+            if (arguments.get("syscalls") != "all")
+            {
+                std::stringstream s;
+                s << arguments.get("syscalls");
+
+                std::string requested_syscall;
+                while (std::getline(s, requested_syscall, ','))
+                {
+                    const auto& syscall_nr = std::find_if(
+                        cbegin(syscall_names()), cend(syscall_names()),
+                        [&requested_syscall](const std::pair<int64_t, std::string>& syscall)
+                        { return syscall.second == requested_syscall; });
```

Comment:
We could be nice here and allow users to also give syscall numbers. std::from_chars is probably your friend here.